src/printer.js

```diff
@@ -414,7 +414,8 @@ const embed = (path, print, textToDoc, opts) => {
     // the css parser appends an extra indented hardline, which we want outside of the `indent()`,
     // so we remove the last element of the array
     const [formatttedStyles, _] = textToDoc(node.content.styles, { ...opts, parser: parserLang });
-    return group([styleLang !== 'css' ? `<style lang="${styleLang}">` : '<style>', indent([hardline, formatttedStyles]), hardline, '</style>', hardline]);
+    const globalAttr = node.attributes.filter((x) => x.name === 'global').length ? ' global' : '';
```

Comment:
@antonyfaris I think we can make an array of which attributes are meant to be kept, to retain modularity. What do you think?

Reply:
You are right my solution wasn't modular. According to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/style there are specific attributes for the style tag and there are [global attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes). So either we make an array or just keep all the original attributes.